netengine/backends/snmp/base.py

```diff
@@ -94,7 +95,26 @@ def _oid(self, oid):
         # ensure is string (could be unicode)
         return str(oid)
 
-    def get(self, oid):
+    def walk(self, oid):
+        result = dict()
```

Comment:
Why are you not directly creating Trie?

Reply:
This was a mistake, thanks I've changed it.